modules/storage.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+import Books from './bookClass.js';
+import { render } from './render.js';
```

Comment:
- [ ] As I mentioned in some of the previous comments, make sure you fix your linters errors instead of disabling them. This for best practices and clean code